### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CreateDatabaseMasterKey_ColorContrast: Color contrast ratio of "Preview button" for normal theme is below the expected ratio i.e 4.5:1

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5751]
[“Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)  - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.3; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:** 
Application name: Azure Data Studio 
 Version: 1.8.0-insider (user setup)
 Electron: 3.1.8
 Node.js: 10.2.0
 V8: 6.6.346.32
 OS: Windows_NT x64 10.0.17763

**Additional Details**
Tool Name: ColorContrast
MAS Reference:MAS1.4.3

**Repro Steps:**  
1.Launch Visual Studio code desktop application.
2.Navigate to "Enable SQL server2019 preview->select table from the database->Create External Table screen should open->click on next button.
3.Navigate to Setting button of Activity Bar and select "Color Theme" option.
4.Open color contrast analyser and check the color contrast ratio.
5.It shows color contrast ratio less than expected color contrast ratio.

**Actual:**
 Color contrast ratio for "Preview", "Next", and "Cancel" buttons is showing as 3.5:1 for normal theme.

**Same issue observed for below page:**
**1.CreateConntionDataSrc**: Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.
**2.CSVModifyColumn:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.

**Expected:**
The expected color contrast ratio for "Preview", "Next", and "Cancel" buttons should be greater than or equal to 4.5:1 ratio.

**User Impact:**
Low vision User will find it difficult to understand and will be confused if they will not able to see the buttons on this page.

**Recommendations:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.4.3-** https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D

**Attachment For Reference:**

![pwd](https://user-images.githubusercontent.com/39204909/58681863-fd9b0a80-838b-11e9-87f2-2c130e7d386e.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Similar to issue : #6667
Should be fixed in https://github.com/microsoft/azuredatastudio/pull/6774

**Comment by chlafreniere** [comment]
As @udeeshagautam mentioned the issue has been fixed since this bug was opened. Closing.

![image](https://user-images.githubusercontent.com/40371649/67136527-4c7efa80-f1dc-11e9-9813-f8b32370c930.png)
